.ci/travis/travis.sh

```diff
@@ -169,8 +169,8 @@ assembly-run-all-jar)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo version:$CS_POM_VERSION
-  FOLDER=src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap
-  FILE=InputNoLineWrapGood.java
+  FOLDER=src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks
+  FILE=InputRightCurlyAloneOrEmptyNoViolations.java
```

Comment:
I am not seeing why this changed.

Reply:
it is still in use by `it`:
```
✔ ~/java/github/romani/checkstyle [6807-suppress-rightcurlyalone L|✔] 
$ ag "InputNoLineWrapGood" src/it/java
src/it/java/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/NoLineWrapTest.java
57:        final String filePath = getPath("InputNoLineWrapGood.java");
```

I just create new file that is located in `it`, and not used by `it` but used by CI.
Ones we create new Check, this file become used by `it` and `CI`.

Reply:
I'm not understanding. What is the purpose of this file / folder assignment if it can change and have no impact on CI? Why was old file needed before and not now?

Reply:
it does affect CI.
you can change this file to any other Input and CI will fail, as it is very rare Input in our google ITs have some code and completely follow google style. So Input is specially selected.
Old file was used as it produce no violations, it is still good to be used in this CI item.

BUT I introduced new Input file that is not used in ITs at all, as it is for future Check, so I reused it in this CI item to be used at least by something and alarm us if somebody try to change suppression in wrong way.

Reply:
> very rare Input in our google ITs have some code and completely follow google style. So Input is specially selected.

Just a thought for another time, but why not keep the file the same and just build a super file that contains everything and not need to switch it from file to file.